database/migrations/2017_01_31_112704_add_parent_key_result_to_objectives.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddParentKeyResultToObjectives extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
__[phpmd]__
* Avoid using short method names like AddParentKeyResultToObjectives::up(). The configured minimum method name length is 3.

<!-- Comment by [SideCI](https://sideci.com) -->